blueos_repository/consolidate.py

```diff
@@ -58,6 +73,14 @@ class Version:
     readme: Optional[str]
     company: Optional[Company]
     support: Optional[str]
+    type: ExtensionType
+    filter_tags: List[str]
+    extra_links: Dict[str, str]
+
+    @staticmethod
+    def validate_filter_tags(tags: List[str]) -> List[str]:
+        """Returns a list of up to 10 lower-case alpha-numeric tags (dashes allowed)."""
+        return [tag.lower() for tag in tags if tag.replace("-", "").isalnum()][:10]
```

Comment:
We should enforce our defined tags and not allow custom ones.

```python
return [tag for tag in tags if tag in (str(e) for e in ExtensionTags)]
```

Reply:
We don't have fixed tags, just fixed types. If someone wants to create an arbitrary tag then they can.

Reply:
So... I can have a tag that will show on the interface as `Potato`, `witchcraft`, emojis, non latin characters and etc ?

Reply:
No, they're required to be lower-case alpha-numeric tags (with dashes allowed), and there's a limit of 10 per extension, as specified in the docstring of that function, as well as in our extensions metadata documentation and in the readme as well.